frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-wizard/configure.tsx

```diff
@@ -76,14 +76,18 @@ export const EncryptionFormGroup: React.FC<EncryptionFormGroupProps> = ({
   }, [encryption.clusterWide, encryption.storageClass, encryptionChecked]);
 
   const toggleEncryption = (checked: boolean) => {
-    setEncryptionDispatch(ActionType.SET_ENCRYPTION, mode, dispatch, {
+    const payload = {
       ...encryption,
       clusterWide: checked,
-    });
-    setEncryptionChecked(checked);
+    };
     if (!checked) {
+      payload.advanced = false;
+      payload.hasHandled = false;
```

Comment:
That seems weird but it is used for validation and need to be set to `true`
```suggestion
      payload.hasHandled = true;
```

Reply:
let me check this

Reply:
The initial state sets "hasHandled" to true, lets follow the same defaults.